Braze documentation updates (#1112)
rudderlabs/rudderstack-docs: docs/destinations/streaming-destinations/braze.mdx

});
```

### Nested array operations

You can also send nested arrays as the `track` properties and perform create, add, update, and remove operations on them. To do so, you must first enable the **Nested Array Operations** dashboard setting. Then, include the field `nestedOperationType` in the `track` event, as shown:

```javascript
rudderanalytics.track("Product Viewed", {
  "nestedOperationType": "update",
  "pets": [{
      "identifier_key": "id",
      "identifier_value": 4,
      "new_object": {
        "breed": "fish"
      }
    },
    {
      "identifier_key": "id",
      "identifier_value": 5,
      "new_object": {
        "name": "alex"
      }
    }
  ]
});
```

> Note that the `nestedOperationType` field is required for this feature to work and must be one of `create`, `update`, `add`, or `remove`.

Reviewer: ```suggestion
Note that `nestedOperationType` is a required field for this feature and must take one of the `create`, `update`, `add`, or `remove` values.
```
